app/a/Makefile

```diff
@@ -0,0 +1,21 @@
+TARGET=a.bin
+OBJS=a.o syscall.o
```

Comment:
please use a Makefile similar to app/ping/Makefile that uses app/liumlib/syscall.S. (Only NAME= parameter needs to be modified.)